test/Validation/MissionSequence.cpp

```diff
@@ -0,0 +1,96 @@
+/// Apache License 2.0
+
+#include <Validation/MissionSequence.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace validation
+{
+
+MissionSequence::MissionSequence(const Dictionary& aDataTree)
+    : dataTree_(aDataTree),
+      initialStateAndSatelliteSystem_(Parser::CreateInitialStateAndSatelliteSystem(dataTree_)),
```

Comment:
I would separate these into two separate methods:
- Parser::ParseInitialState
- Parser::ParseSatelliteSystem

Reply:
It would be cleaner that way, but the reason I combined them is that they constructing those objects is quite related and is more easily done together

Reply:
actually it is cleaner that way I made the change